README.md

```diff
@@ -46,6 +46,10 @@ If you encounter the following error "Import-Module : ...psake.psm1 cannot be lo
 
 If you wish to use the psake command from outside of the install folder, add the folder install directory to your PATH variable.
 
+**Step 5: (With VS2017)** Install the VSSetup dependency
+
+Psake uses [VSSetup](https://blogs.msdn.microsoft.com/heaths/2017/01/25/visual-studio-setup-powershell-module-available/) to locate msbuild when using Visual Studio 2017.  The VSSetup PowerShell module must be installed prior to compiling a VS2017 project with psake.  Install instructions for VSSetup can be found [here](https://github.com/Microsoft/vssetup.powershell#installing) and [here](https://www.powershellgallery.com/packages/VSSetup).

```

Comment:
My only suggestion is that other parts of the readme use `psake` (all lower case) even when starting a sentence. Other than that, looks good.